src/core/public/doc_links/doc_links_service.ts

```diff
@@ -344,12 +344,38 @@ export interface DocLinksStart {
     readonly ml: Record<string, string>;
     readonly transforms: Record<string, string>;
     readonly visualize: Record<string, string>;
-    readonly apis: Record<string, string>;
+    readonly apis: {
```

Comment:
__note:__ it's very easy to make a typo in a link name and with `Record<string, string>` signature it may go unnoticed. Changing signatures only for those that we use in Security for now.

Reply:
This is much better! Makes it easier to know what links are available as well with autocomplete. 

The properties you've added don't have the readonly flag. Might be worth just wrapping the whole interface in `Readonly<T>` utility type since literally everything should be marked as readonly. 
https://www.typescriptlang.org/docs/handbook/utility-types.html#readonlytype

Reply:
Ah, good catch! Will do